docs/fr/users/04_Subscriptions.md

```diff
@@ -1,19 +1,23 @@
 # Ajouter un flux
 
-**TODO**
+**À FAIRE**
 
 # Import et export
 
-**TODO**
+**À FAIRE**
 
-# Utiliser le « bookmarklet »
+# Utiliser le « bookmarklet »
 
-Les « bookmarklets » sont de petits scripts que vous pouvez exécuter pour effectuer des tâches utiles ou frivoles. FreshRSS offre un signet ( « bookmark » ) pour s'abonner aux fils de nouvelles.
+Les « bookmarklets » sont de petits scripts que vous pouvez exécuter pour
+effectuer des tâches utiles ou frivoles. FreshRSS offre un signet ( «
```

Comment:
effectuer des tâches diverses et variées. FreshRSS offre un signet ( «

Reply:
lol, @Alkarex didn't like this one either. I'll change the English too.

Are you purposefully also going through old stuff btw? No objections of course, but as you can see in the diff all that happened here is that po4a ended up adding a little line break.

Reply:
I go through all files in french. No matter old or new. I think it's a good time to do it since you're doing your part to maintain the english version. By teaming up, we will be faster :)